### PR TITLE
Review summary minor visual fixes

### PR DESCRIPTION
**Pull request by pandafulmanda**
screen shot 2015-08-14 at 1 52 51 pm

## Without fix
screen shot 2015-08-17 at 12 46 39 pm

See openstax/tutor-server#585 for period name being too long

### CODE REVIEW

File: src/components/course-periods-nav.cjsx [hunk 1]
```diff
@@ -56,7 +61,10 @@ CoursePeriodsNav = React.createClass
     @setState(active: key)
 
   renderPeriod: (period, key) ->
-    <BS.NavItem eventKey={key} key="period-nav-#{period.id}">{period.name}</BS.NavItem>
+    {name, id} = period
+    name = name.replace(' Period', '') if @props.format is 'short' and name?
```
Comment (philschatz) [inline src/components/course-periods-nav.cjsx]: Could you change the title in the demo script to be 4chars?

Comment (pandafulmanda) [inline src/components/course-periods-nav.cjsx]: OH in the BE, i think so, yeah

**Comment by pandafulmanda** [comment]
@philschatz i reverted the string replace periods name stuff

**Comment by nathanstitt** [comment]
Looks good!

**Pull request merged**